customized-greeting-cards-using-openai/service.bal

```diff
@@ -0,0 +1,68 @@
+import ballerina/http;
+import ballerinax/googleapis.gmail;
+import ballerinax/openai.dalle; // the names will change in th final connectors
+import ballerinax/openai.gpt3; // the names will change in th final connectors
+
+configurable string openAIToken = ?;
+configurable string gmailToken = ?;
+
+final gpt3:Client gpt3Client = check new ({auth: {token: openAIToken}});
+final dalle:Client dalleClient = check new ({auth: {token: openAIToken}});
+final gmail:Client gmailClient = check new ({auth: {token: gmailToken}});
+
+type Reqquest record {|
+    string occasion;
+    string recipientEmail;
+    string emailSubject;
+    string specialNotes?;
+|};
+
+service / on new http:Listener(8080) {
+    resource function post greetingCard(@http:Payload Reqquest req) returns error? {
+
+        string occasion = req.occasion;
+        string recipientEmail = req.recipientEmail;
+        string emailSubject = req.emailSubject;
+        string specialNotes = req.specialNotes ?: "";
+
+        fork {
```

Comment:
We added this since these two sub-tasks can be done in parallel and we can also exhibit Ballerina's concurrent execution with this sample. However, if it is not recommended for this use case, we can remove it.

Reply:
I meant just the fork statement, we can still have the workers without a fork statement.